bedrock/bedrock-common/src/main/java/com/nukkitx/protocol/bedrock/data/command/CommandParam.java

```diff
@@ -0,0 +1,62 @@
+package com.nukkitx.protocol.bedrock.data.command;
+
+import com.nukkitx.protocol.bedrock.BedrockPacketHelper;
+
+public class CommandParam {
```

Comment:
Why not to use enum and why not to use Lombok for Getters, toString etc.

Reply:
Because we create `CommandParam` dynamically. We use the static one for known param ids, when we do not know the param type (which is enum) we create new `CommandParam`  which holds its inv value.

I dont see here big advantages of using Lombok. I would still have to create two constructors manually. I also wanted to print `UNKNOWN` when `paramType` is null so `@ToString` wouldnt help.

Reply:
Thx for clarification.
But I still think @Getter or @RequierdArgsConstructor will suit that class.
In my opinion less repetitive code is better.

Reply:
Sure, I can change it.